src/MizarData/ActiveFunctionTimePerFrameComparatorTest.cpp

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock-matchers.h>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <cstdint>
+#include <limits>
+#include <type_traits>
+#include <utility>
+
+#include "MizarData/ActiveFunctionTimePerFrameComparator.h"
+#include "MizarData/BaselineOrComparison.h"
+#include "MizarData/SampledFunctionId.h"
+#include "MizarData/SamplingWithFrameTrackComparisonReport.h"
+
+namespace orbit_mizar_data {
+
+using testing::DoubleNear;
+using testing::IsNan;
+using testing::Return;
+
+namespace {
+class MockCounts {
+ public:
+  MOCK_METHOD(double, GetExclusiveRate, (SFID), (const));
+  MOCK_METHOD(uint64_t, GetTotalCallstacks, (), (const));
+};
+
+class MockFrameTrackStats {
+ public:
+  MOCK_METHOD(double, ComputeAverageTimeNs, (), (const));
+  MOCK_METHOD(uint64_t, variance_ns, (), (const));
+  MOCK_METHOD(uint64_t, count, (), (const));
+};
+
+}  // namespace
+
+constexpr uint64_t kTotalCallstacksBaseline = 100;
+constexpr uint64_t kTotalCallstacksComparison = 200;
+
+constexpr double kRateBaseline = 0.1;
+constexpr double kRateComparison = 0.15;
+
+constexpr uint64_t kFramesCountBaseline = 1000;
+constexpr uint64_t kFramesCountComparison = 1300;
+
+constexpr double kFrametimeVarBaseline = 100;
+constexpr double kFrametimeVarComparison = 150;
+
+constexpr double kAvgFrametimeBaseline = 1000;
+constexpr double kAvgFrametimeComparison = 900;
+
+constexpr SFID kArbitrarySfid(10);
+
+constexpr double kExpectedStatistic = -0.929944;
+constexpr double kPvalue = 0.352400;
+constexpr double kTolerance = 1e-3;
+
+class ActiveFunctionTimePerFrameComparatorTest : public ::testing::Test {
+ public:
+  ActiveFunctionTimePerFrameComparatorTest()
+      : baseline_counts_(std::in_place_t{}),
+        comparison_counts_(std::in_place_t{}),
+        baseline_frame_track_stats_(std::in_place_t{}),
+        comparison_frame_track_stats_(std::in_place_t{}) {}
```

Comment:
You should use `std::in_place` instead of `std::in_place_t{}`. It is a global constant (an instance of `std::in_place_t`) which exists for the purpose to pass it in as a tag.